tests/pytest.ini

```diff
@@ -1,3 +1,4 @@
 [pytest]
+addopts = -v -rsXx
```

Comment:
Sets the default commandline options for pytest so that we don't have to specify it multiple places.

Reply:
👍